src/Web/Sfa.Das.Sas.Web.UnitTests/Infrastructure/Web/Controllers/SitemapControllerTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections.Generic;
+using System.Web;
+using System.Web.Mvc;
+using System.Web.Routing;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Sfa.Das.Sas.Web.Controllers;
+using Sfa.Das.Sas.Web.Helpers;
+using Sfa.Das.Sas.Web.Services;
+
+namespace Sfa.Das.Sas.Web.UnitTests.Infrastructure.Web.Controllers
+{
+    [TestClass]
```

Comment:
Could you use nunit here and remove the reference to Microsoft.VisualStudio.TestTools.UnitTesting